resources/lib/__init__.py

```diff
@@ -0,0 +1,12 @@
+from .login import login, logout
+from .ui.router import route
+
+
+@route('wonderful_login')
```

Comment:
Why those are here and not under default?

Reply:
Good point. Didn't see default.py, so I'll move it there!